### PR TITLE
Submit feedback based on thumb click

### PR DESCRIPTION
**Pull request by dumez-k**
In order to increase engagement and make it as easy as possible for people to submit feedback, upon thumbs up or thumbs down feedback will be forwarded with the option afterwards to add more comments and resubmit.

### CODE REVIEW

File: components/Feedback/Feedback.tsx [hunk 1]
```diff
@@ -44,13 +30,37 @@ export default function PageWithJSbasedForm(props) {
 
     //Send feedback to posthog
     void sendDocsFeedback(feedback, comment);
+  };
+
+  const handleSubmit = (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
 
+    if (!feedback) {
+      alert("Please select a feedback!");
```
Comment (benarent) [inline components/Feedback/Feedback.tsx:39]: is this just from testing?

Comment (dumez-k) [reply components/Feedback/Feedback.tsx:39]: Also if someone bypasses the thumbs to get to the submit button early. An alert will pop up telling them they can't do that

Comment (dumez-k) [reply components/Feedback/Feedback.tsx:39]: like using the chrome dev console

Comment (benarent) [reply components/Feedback/Feedback.tsx:39]: Can we make it console log in that case, I'm not a fan of shipping alert to production sites

Comment (dumez-k) [reply components/Feedback/Feedback.tsx]: on second look I think it's ok to remove, I wasn't able to replicate the behavior I described above 👍

Comment (dumez-k) [reply components/Feedback/Feedback.tsx]: Removed

**Review by benarent — APPROVED** [review]

**Pull request merged**